source/_components/image_processing.openalpr_local.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "OpenAlpr Local"
+description: "Instructions how to integrate licences plates with OpenAlpr local into Home Assistant."
+date: 2017-01-09 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: openalpr.png
+ha_category: Image_Processing
+featured: false
+ha_release: 0.36
+---
+
+[OpenAlpr](http://www.openalpr.com/) integration for Home Assistant allows you to process licences plates from a camera. You can use them to open a garage door or trigger any other [automation](https://home-assistant.io/components/automation/).
+
+### {% linkable_title Local installation %}
+
+If you want process all data local you need the command line tool `alpr` in version > 2.3.1
```

Comment:
s/local/locally/

"…you need version 2.3.1 or higher of the `alpr` command line tool"